common/constants.go

```diff
@@ -13,6 +13,7 @@ import (
 
 const (
 	DefaultImageRegistry = "netappdownloads.jfrog.io/docker-astra-control-staging/arch30/neptune"
+	AstraImageRegistry   = "cr.astra.netapp.io"
```

Comment:
@mgunaseg plz confirm there aren't any subdirectories, e.g. "neptune" in the ACS repo?
otherwise we might need this to be "cr.astra.netapp.io/neptune"